src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-form/rgw-user-form.component.ts

```diff
@@ -113,6 +113,7 @@ export class RgwUserFormComponent implements OnInit {
         ]
       ]
     });
+    this.userForm._filterValue = (value) => value;
```

Comment:
Please add a comment here why it is necessary to override this function.

Reply:
IMHO, a comment on that function doesn't make the implementation better. What we really need, and that's what you've already said @votdev, is a refactoring.